cypress/integration/tests/rbac/architect.test.ts

```diff
@@ -1,12 +1,40 @@
 import { User } from "../../models/keycloak/users/user";
-import { getRandomUserData } from "../../../utils/data_utils";
+import { getRandomCredentialsData, getRandomUserData } from "../../../utils/data_utils";
 import { UserArchitect } from "../../models/keycloak/users/userArchitect";
-import { preservecookies } from "../../../utils/utils";
+import {
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+    logout,
+    preservecookies,
+} from "../../../utils/utils";
+import { Analysis } from "../../models/developer/applicationinventory/analysis";
+import { CredentialsSourceControlUsername } from "../../models/administrator/credentials/credentialsSourceControlUsername";
+import { CredentialType } from "../../types/constants";
 
 describe("Architect RBAC operations", () => {
     let userArchitect = new UserArchitect(getRandomUserData());
+    const application = new Analysis(
+        getRandomApplicationData("bookServerApp"),
+        getRandomAnalysisData({
```

Comment:
You can load this data using 


       const application = new Analysis(
            getRandomApplicationData("bookserverApp", { sourceData: this.appData[0] }),
            getRandomAnalysisData(this.analysisData[0])
        );  




Example : https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/integration/tests/developer/applicationinventory/analysis/source_analysis.test.ts#L101

Reply:
Tried and it didn't work, I will try again